src/renderer/components/outputs/OutputContainer.tsx

```diff
@@ -1,63 +1,172 @@
-                    <div style={{ position: 'absolute', right: '-4px', width: 0 }}>
-                        <Handle
-                            isConnectable
+                    <Center
+                        position="absolute"
+                        right="-6px"
+                    >
+                        <Div
+                            _before={{
+                                content: '" "',
+                                position: 'absolute',
+                                top: '50%',
+                                left: '50%',
+                                height: '45px',
+                                width: '45px',
+                                cursor: 'crosshair',
+                                // backgroundColor: '#FF00FF1F',
+                                transform: 'translate(-50%, -50%)',
+                                borderRadius: '100%',
+                            }}
+                            _hover={{
+                                width: '22px',
+                                height: '22px',
+                                marginRight: '-3px',
+                                opacity: showHandle ? 1 : 0,
+                            }}
+                            as={RightHandle}
+                            className="output-handle"
                             id={`${id}-${outputId}`}
                             isValidConnection={isValidConnection}
-                            position={Position.Right}
-                            style={{
-                                width: '15px',
-                                height: '15px',
-                                borderWidth: '1px',
-                                borderColor,
-                                transition: '0.25s ease-in-out',
-                                background: handleColor,
+                            sx={{
+                                width: '16px',
+                                height: '16px',
+                                borderWidth: '2px',
+                                borderColor: handleColor,
+                                transition: '0.15s ease-in-out',
+                                background: isConnected ? connectedColor : handleColor,
+                                boxShadow: '-2px 2px 2px #00000014',
+                                filter: showHandle ? undefined : 'grayscale(100%)',
+                                opacity: showHandle ? 1 : 0.3,
+                                position: 'relative',
                             }}
-                            type="source"
                             onContextMenu={noContextMenu}
                         />
-                    </div>
+                    </Center>
                 </HStack>
             );
         }
 
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        const [gray300, gray700]: string[] = useToken('colors', ['gray.300', 'gray.700']);
```

Comment:
Please use `useToken() as string[]`. ESLint won't complain that way.

Same for the others `useToken()`s as well.